move.c

```diff
@@ -70,33 +58,15 @@ int32_t required_micrometers_to_speed(float speed)
 }
 
 /**
- * @brief Calculate the required time to reach a given speed, in seconds.
- *
- * This functions assumes the current speed is the target speed and takes into
- * account the configured linear deceleration.
- */
-float required_time_to_speed(float speed)
```

Comment:
I would rather not remove these `required_*()` functions, even if they are currently not used.

They were tested and we may need them in the future! :blush:

Reply:
483b200  updated